### ISSUE TITLE
OpenAI API coming up as invalid

### ISSUE THREAD

**Issue by Aryc7** [brianpetro/obsidian-smart-connections#468]
Anytime I put in my API key it shows up as error

![Screenshot 2024-02-17 at 5 54 52 PM](https://github.com/brianpetro/obsidian-smart-connections/assets/39039416/d11e4287-978d-4170-87f0-e6ceb4376975)

**Comment by brianpetro** [comment]
Hey @Aryc7 

It looks like there were some issues with the API early. Is it still happening?

![Screenshot_20240217-231523](https://github.com/brianpetro/obsidian-smart-connections/assets/1886014/8b6144d7-3a21-457a-873d-6372c46346e8)

**Comment by david-triba** [comment]
I'm experiencing this as well. Even today..

**Comment by Aryc7** [comment]
Yes - still having this problem.

**Comment by brianpetro** [comment]
@Aryc7 @david-triba checkout this thread and let me know how it goes https://github.com/brianpetro/obsidian-smart-connections/discussions/155#discussioncomment-7632050

**Comment by Tiriyon** [comment]
Similar issue here

**Comment by david-triba** [comment]
@brianpetro Yes, the issue for me was that my balance was on $0.00. As soon as I added credits, it worked.
